jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/handler/WildFlyAppSeverHandlerTest.java

```diff
@@ -17,155 +17,131 @@
-    }};
-    // @formatter:on
+    when(generatorContext.getRuntimeMode()).thenReturn(RuntimeMode.OPENSHIFT);
+    when(generatorContext.getStrategy()).thenReturn(JKubeBuildStrategy.docker);
     // When
     final WildFlyAppSeverHandler handler = new WildFlyAppSeverHandler(generatorContext);
     // Then
     assertCommonValues(handler);
-    assertEquals("jboss/wildfly:25.0.0.Final", handler.getFrom());
-    assertEquals("/opt/jboss/wildfly/standalone/deployments", handler.getDeploymentDir());
-    assertEquals(Collections.singletonList("chmod -R a+rw /opt/jboss/wildfly/standalone/"), handler.runCmds());
+    assertThat(handler.getFrom()).isEqualTo("jboss/wildfly:25.0.0.Final");
+    assertThat(handler.getDeploymentDir()).isEqualTo("/opt/jboss/wildfly/standalone/deployments");
+    assertThat(handler.runCmds()).isEqualTo(Collections.singletonList("chmod -R a+rw /opt/jboss/wildfly/standalone/"));
   }
 
   @Test
   public void openShiftSourceStrategy() {
     // Given
-    // @formatter:off
-    new Expectations() {{
-      generatorContext.getRuntimeMode(); result = RuntimeMode.OPENSHIFT;
-      generatorContext.getStrategy(); result = JKubeBuildStrategy.s2i;
-    }};
-    // @formatter:on
+    when(generatorContext.getRuntimeMode()).thenReturn(RuntimeMode.OPENSHIFT);
+    when(generatorContext.getStrategy()).thenReturn(JKubeBuildStrategy.s2i);
     // When
     final WildFlyAppSeverHandler handler = new WildFlyAppSeverHandler(generatorContext);
     // Then
     assertCommonValues(handler);
-    assertEquals("quay.io/wildfly/wildfly-centos7:26.0", handler.getFrom());
-    assertEquals("/deployments", handler.getDeploymentDir());
-    assertTrue(handler.runCmds().isEmpty());
+    assertThat(handler.getFrom()).isEqualTo("quay.io/wildfly/wildfly-centos7:26.0");
+    assertThat(handler.getDeploymentDir()).isEqualTo("/deployments");
+    assertThat(handler.runCmds()).isEmpty();
   }
 
   private static void assertCommonValues(WildFlyAppSeverHandler handler) {
-    assertTrue(handler.supportsS2iBuild());
-    assertEquals("deployments", handler.getAssemblyName());
-    assertEquals("/opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0", handler.getCommand());
-    assertEquals("jboss:jboss:jboss", handler.getUser());
-    assertEquals(Collections.singletonMap("GALLEON_PROVISION_LAYERS", "cloud-server,web-clustering"), handler.getEnv());
-    assertEquals(Collections.singletonList("8080"), handler.exposedPorts());
+
+    assertThat(handler.supportsS2iBuild()).isTrue();
+    assertThat(handler.getAssemblyName()).isEqualTo("deployments");
+    assertThat(handler.getCommand()).isEqualTo("/opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0");
+    assertThat(handler.getUser()).isEqualTo("jboss:jboss:jboss");
+    assertThat(handler.getEnv()).isEqualTo(Collections.singletonMap("GALLEON_PROVISION_LAYERS", "cloud-server,web-clustering"));
+    assertThat(handler.exposedPorts()).isEqualTo(Collections.singletonList("8080")).isEqualTo( handler.exposedPorts());
```

Comment:
This doesn't look right

```suggestion
    assertThat(handler.exposedPorts()).isEqualTo(Collections.singletonList("8080"));
```